phpunit.xml.dist

```diff
@@ -17,4 +17,7 @@
             <directory>./src/</directory>
         </include>
     </coverage>
+    <php>
+        <ini name="error_reporting" value="-1" />
+    </php>
```

Comment:
The tests for PHP 8.2 and PHP 8.3 are currently running infinitely and I narrowed it down to these three lines being the cause of this. For other pull requests these changes ran smoothly, any idea what is happening here @clue?

Reply:
This looks strange. Can you reproduce this locally? Can you try excluding some tests to see which test is causing this? Using groups or excludes or filters might be useful.

Reply:
Hello @clue, I tried debugging the code to find the cause of this issue and after seeing a new error thrown in line 76 in the `CompositeConnection.php` file I ended up looking in the changes made in PHP 8.2 and saw that the dynamic properties are deprecated, see https://www.php.net/releases/8.2/en.php#deprecate_dynamic_properties. I decided to add a new class variable `private &remote` after discussion the new situation with @SimonFrings. Tests are running green now. 👍

Reply:
I also updated my comment and explain the situation. Thank you @clue for giving me the hint to exclude groups. It was definitely faster for narrowing it down. 👍